server/control.go

```diff
@@ -266,15 +266,15 @@ func (c Controller) runOnce(ctx context.Context, leaderKey string, tick <-chan t
 		return err
 	}
 
-	reboot, err := inf.Storage().GetRebootsFrontEntry(ctx)
-	switch err {
-	case nil:
-	case cke.ErrNotFound:
-		reboot = nil
-	default:
+	re, err := inf.Storage().GetRebootsEntries(ctx)
+	if err != nil {
 		return err
 	}
-
+	metrics.UpdateReboot(len(re))
```

Comment:
IIUC, this metric should also be updated every time the server removes an entry from the queue.